.github/workflows/binary.yaml

```diff
@@ -104,6 +104,7 @@ jobs:
           cp result/bin/dune ~/build/$ARCHIVE_DIR
           cp -r $GITHUB_WORKSPACE/dune-binary-distribution/env ~/build/$ARCHIVE_DIR
           cp -r $GITHUB_WORKSPACE/dune-binary-distribution/completions ~/build/$ARCHIVE_DIR
+          cp -r $GITHUB_WORKSPACE/dune-binary-distribution/tool-wrappers ~/build/$ARCHIVE_DIR
```

Comment:
I'm thinking, wouldn't it be nicer to install the ocamllsp wrapper script as part of the install script instead of the tarball? That way it can be updated without having to wait for a nightly build and can be updated separately from the tarballs.

Reply:
I don't see the wrapper script being changed too frequently, and I kind of like all the files we install are in the tarball. It would complicate the install process if the install script downloaded additional files like the wrapper script and installed them separately from the contents of the tarball. Also if we do want to quickly update the script in the nightly build we can always manually rerun the pipeline.